Extension/CHANGELOG.md

```diff
@@ -1,79 +1,60 @@
 # C/C++ for Visual Studio Code Changelog
 
-## Version 1.12.2 (pre-release): August 24, 2022
-## Enhancements
+## Version 1.12.3: August 29, 2022
+### New Features
+* Enable generate Doxygen comment through context menu, code action and typying. [#5683](https://github.com/microsoft/vscode-cpptools/issues/5683)
```

Comment:
Remove "enable" and just say "Generate Doxygen...". "Enable" makes it sound like it was there before but this is a new feature.

Reply:
Remove "enable" from the enhancements items too.

Reply:
Oh I see -- the 1.12.1 has the upated comments -- I was using old ones.

Reply:
I've re-worded the Enable enhancements as well.